src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="#"
+    method="post"
+    class="input-form"
+    data-qa="big"
+    >
+      <label for="search-input-big">
```

Comment:
`label` is redundant

Reply:
I'm sorry, I don't understand this comment. Should I replace the label at all? By what tag?

Reply:
I replaced by div, I hope it's OK :)